gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/sampling/PageRankSamplingTest.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling;
+
+import com.google.common.collect.Lists;
+import org.apache.flink.api.java.io.LocalCollectionOutputFormat;
+import org.gradoop.common.model.impl.pojo.Edge;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class PageRankSamplingTest extends ParametrizedTestForGraphSampling {
+
+  public PageRankSamplingTest(String testName, String seed, String sampleSize,
+    String dampeningFactor, String maxIteration, String sampleGreaterThanThreshold,
+    String keepVerticesIfSameScore) {
+    super(testName, Long.parseLong(seed), Float.parseFloat(sampleSize),
+      Double.parseDouble(dampeningFactor), Integer.parseInt(maxIteration),
+      Boolean.parseBoolean(sampleGreaterThanThreshold),
+      Boolean.parseBoolean(keepVerticesIfSameScore));
+  }
+
+  /**
+   * {@inheritDoc}
```

Comment:
inheritance not needed with @Override if no additions were made by doc. Same below.